views/pages/layout.ejs

```diff
@@ -40,6 +40,9 @@
       <link rel="preload" href="<%= jsPath %>" as="script">
     <%_ }) _%>
     <%_ } _%>
+    <% if (isOnesignalEnable && oneSignalAppId) { %>
```

Comment:
if the enable is set as false by default then we will have to update the docs as well saying we have this config and that they need to set to true if they want onesignal